test/jest/Bladeburner/Actions.test.ts

```diff
@@ -222,15 +209,15 @@ describe("Bladeburner Actions", () => {
 
     describe.each([SampleOperation, SampleBlackOp])("operations and black operations decrease rank", (id) => {
       it(`${id.type}`, () => {
-        guaranteeFailure(), (before = inst.rank), complete(id);
-        expect(inst.rank).toBeLessThan(before);
+        (before = bb.rank), complete(id, forceFailure);
+        expect(bb.rank).toBeLessThan(before);
       });
     });
   });
 
   it("have a minimum duration of 1 second", () => {
     complete(SampleContract);
-    expect(inst.actionTimeToComplete).toBeGreaterThanOrEqual(1);
+    expect(bb.actionTimeToComplete).toBeGreaterThanOrEqual(1);
   });
 
   beforeAll(() => {
```

Comment:
Optional: IMO it flows better if your setup code is moved to before the test code, but up to you.